package.json

```diff
@@ -13,7 +13,7 @@
   "repository": "https://github.com/irrationalistic/atom-tasks",
   "license": "MIT",
   "engines": {
-    "atom": ">0.50.0"
+    "atom": ">= 1.13"
```

Comment:
This is recommended in the Atom [blog post about this deprecation](http://blog.atom.io/2016/11/14/removing-shadow-dom-boundary-from-text-editor-elements.html).

> Please note that once you change your style sheets to remove the deprecated shadow DOM selectors, they will no longer be compatible with versions of Atom prior to 1.13. As such, we recommend that package and theme authors set the Atom version to >= 1.13 in the engines section of their packages’ package.json to prevent the upgraded packages from being installed on older versions of Atom.